### PR TITLE
Added and fixed some tests.

### PR DESCRIPTION
**Pull request by grejucv**

### CODE REVIEW

File: src/test/java/models/testSpanishGame.java [hunk 1]
```diff
@@ -49,10 +49,10 @@ public void testCustomDeal(){
         g.buildDeck();
         g.customDeal(0,3,6,9);
         System.out.println("Custom Deal: " + g.cols.get(0).get(0).toString() + " " + g.cols.get(1).get(0).toString() + " " + g.cols.get(2).get(0).toString() + " "+ g.cols.get(3).get(0).toString());
-        assertEquals("2Clubs",g.cols.get(0).get(0).toString());
```
Comment (DrewHamm) [inline src/test/java/models/testSpanishGame.java]: Does this remove our old tests for the original game?

File: src/test/java/models/testSpanishGame.java [hunk 2]
```diff
@@ -49,10 +49,10 @@ public void testCustomDeal(){
         g.buildDeck();
         g.customDeal(0,3,6,9);
         System.out.println("Custom Deal: " + g.cols.get(0).get(0).toString() + " " + g.cols.get(1).get(0).toString() + " " + g.cols.get(2).get(0).toString() + " "+ g.cols.get(3).get(0).toString());
```
Comment (DrewHamm) [inline src/test/java/models/testSpanishGame.java:51]: Does this remove our old tests for the original game?

**Comment by DrewHamm** [comment]
Looks like you might have accidentally include some files from the .idea directory. Was that intentional?

File: src/test/java/models/testSpanishGame.java (hunk 2 above)
Comment (grejucv) [inline src/test/java/models/testSpanishGame.java:51]: This is a test for the Spanish deck. The original tests are all still there.

**Pull request merged**